src/url.js

```diff
@@ -179,8 +180,15 @@ export function serializeQueryString(params) {
     const v = params[k];
     if (v == null) {
       continue;
+    } else if (isArray(v)) {
+      for (let i = 0; i < v.length; i++) {
+        const sv = /** @type {string} */ (v[i]);
+        s.push(`${encodeURIComponent(k)}=${encodeURIComponent(sv)}`);
```

Comment:
Is this correct? I thought multi-inputs got turned into `"k[]=v1&k[]=v2"`, not `"k=v1&k=v2"`